src/index.js

```diff
@@ -413,114 +510,171 @@ export class ULabel {
+                let annid =
+                    ul.subtasks[crst]["state"]["idd_associated_annotation"];
+                ul.hide_global_edit_suggestion();
+                ul.show_id_dialog(
+                    ul.get_global_mouse_x(e),
+                    ul.get_global_mouse_y(e),
+                    annid,
+                    false,
+                );
+            },
+        );
 
-        $(document).on("click.ulabel", "#" + ul.config["annbox_id"] + " .delete_suggestion", () => {
-            let crst = ul.state["current_subtask"];
-            ul.delete_annotation(ul.subtasks[crst]["state"]["move_candidate"]["annid"]);
-        })
+        $(document).on(
+            "click.ulabel",
+            "#" + ul.config["annbox_id"] + " .delete_suggestion",
+            () => {
+                let crst = ul.state["current_subtask"];
+                ul.delete_annotation(
+                    ul.subtasks[crst]["state"]["move_candidate"]["annid"],
+                );
+            },
+        );
 
         // Button to save annotations
-        $(document).on("click.ulabel", "#" + ul.config["toolbox_id"] + " a.night-button", function () {
-            if ($("#" + ul.config["container_id"]).hasClass("ulabel-night")) {
-                $("#" + ul.config["container_id"]).removeClass("ulabel-night");
-                // Destroy any night cookie
-                ULabel.destroy_night_mode_cookie();
-            }
-            else {
-                $("#" + ul.config["container_id"]).addClass("ulabel-night");
-                // Drop a night cookie
-                ULabel.set_night_mode_cookie();
-            }
-        })
+        $(document).on(
+            "click.ulabel",
+            "#" + ul.config["toolbox_id"] + " a.night-button",
+            function () {
+                if (
+                    $("#" + ul.config["container_id"]).hasClass("ulabel-night")
+                ) {
+                    $("#" + ul.config["container_id"]).removeClass(
+                        "ulabel-night",
+                    );
+                    // Destroy any night cookie
+                    ULabel.destroy_night_mode_cookie();
+                } else {
+                    $("#" + ul.config["container_id"]).addClass("ulabel-night");
+                    // Drop a night cookie
+                    ULabel.set_night_mode_cookie();
+                }
+            },
+        );
 
         // Keyboard only events
         $(document).on("keydown.ulabel", (keypress_event) => {
             const shift = keypress_event.shiftKey;
             const ctrl = keypress_event.ctrlKey || keypress_event.metaKey;
-            if (ctrl &&
-                (
-                    keypress_event.key === "z" ||
+            if (
+                ctrl &&
+                (keypress_event.key === "z" ||
                     keypress_event.key === "Z" ||
-                    keypress_event.code === "KeyZ"
-                )
+                    keypress_event.code === "KeyZ")
```

Comment:
Maybe the diff isn't making it clear, but this is the output here:
```javascript
if (
    ctrl &&
    (keypress_event.key === "z" ||
        keypress_event.key === "Z" ||
        keypress_event.code === "KeyZ")
) {
```

Not great. Thoughts?